in_text/text_03_config.yml

```diff
@@ -19,21 +19,21 @@ cross-cites:
     pubdate: 2019
     link: https://aed.see.uwa.edu.au/research/models/GLM/configuration.html
 
-
 entities:
   -
     data-name: reservoir_nml_values.json
```

Comment:
Fields in ths json are different than last year. Is there anywhere where I can find info about these attributes?